ProjectSystem.sln

```diff
@@ -72,7 +68,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "items", "items", "{5204CAC5
 		SECURITY.md = SECURITY.md
 	EndProjectSection
 EndProject
-Project("{F184B08F-C81C-45F6-A57F-5ABD9991F28F}") = "Microsoft.VisualStudio.Editors.UnitTests", "tests\Microsoft.VisualStudio.Editors.UnitTests\Microsoft.VisualStudio.Editors.UnitTests.vbproj", "{A0B3F2BD-C92A-4037-A9F0-4EC484267E75}"
+Project("{778DAE3C-4631-46EA-AA77-85C1314464D9}") = "Microsoft.VisualStudio.Editors.UnitTests", "tests\Microsoft.VisualStudio.Editors.UnitTests\Microsoft.VisualStudio.Editors.UnitTests.vbproj", "{A0B3F2BD-C92A-4037-A9F0-4EC484267E75}"
```

Comment:
Is this change to the Microsoft.VisualStudio.Editors.UnitTests GUID intentional?

Reply:
No, I missed this. Will investigate and likely revert. Thanks.

Reply:
These are both VB project type GUIDs:

- F184B08F-C81C-45F6-A57F-5ABD9991F28F Classic (will sniff for SDK style)
- 778DAE3C-4631-46EA-AA77-85C1314464D9 Forced SDK style

I've reverted this particular change. Thanks again.